storage/integration_test.go

```diff
@@ -6637,6 +6712,30 @@ func (h testHelper) mustCreate(b *BucketHandle, projID string, attrs *BucketAttr
 	}
 }
 
+func (h testHelper) mustCreateZonalBucket(b *BucketHandle, projID string) {
+	h.t.Helper()
+
+	// Create a bucket in the same zone as the test VM.
+	zone, err := metadata.ZoneWithContext(context.Background())
+	if err != nil {
+		h.t.Fatalf("could not determine VM zone: %v", err)
```

Comment:
This will cause integration tests to fail locally unless on a VM, correct? Should we add a skip/flag to avoid tests that require this similar to `-short`?